rp2-pio/piolib/i2s.go

```diff
@@ -71,6 +71,14 @@ func (i2s *I2S) WriteStereo(b []uint32) (int, error) {
 	return i2sWrite(i2s, b)
 }
 
+func (i2s *I2S) ReadMono(p []uint16) (n int, err error) {
+	return 0, pio.ErrNotImplemented
```

Comment:
We could instead use `errors.ErrUnsupported` https://pkg.go.dev/errors#pkg-variables